clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -51,15 +52,22 @@ class ServerService private (
     * Returns the number of updates pulled & applied on success.
     */
   private[service] def recoverMetadata[
-    D <: ClioDocument: Decoder: ElasticsearchIndex
-  ](): Future[Int] = {
+    D <: ClioDocument: Decoder
+  ]()(implicit index: ElasticsearchIndex[D]): Future[Int] = {
```

Comment:
Not sure what this means.

Reply:
Moving it to the implicit parameter list gives it a usable identifier within the method (vs having it in the context bound, which pulls it into implicit scope but doesn't enable referring to it directly).